src/components/app-configure/CreateApp.vue

```diff
@@ -105,44 +105,48 @@ async function handleCreateApp() {
     <VCard variant="popup" class="create-app-modal-card">
       <h2 class="create-app-title">Create New App</h2>
       <VSeperator />
-      <VStack direction="column" gap="1rem">
-        <label class="app-name-label">Enter App Name</label>
-        <VTextField
-          v-model="appName"
-          class="app-name-input"
-          :message-type="hasAppNameError ? 'error' : ''"
-          message="App Name cannot be empty"
-        />
-      </VStack>
-      <VStack direction="column" gap="1rem" align="start">
-        <VStack gap="0.5rem">
-          <label class="app-name-label">Choose Region</label>
-          <VTooltip
-            title="Arcana Store uses physical storage nodes that are logically grouped by
+      <form @submit.prevent="handleCreateApp">
+        <VStack direction="column" gap="1rem">
+          <label class="app-name-label" for="app-name">Enter App Name</label>
+          <VTextField
+            id="app-name"
+            v-model.trim="appName"
+            class="app-name-input"
+            :message-type="hasAppNameError ? 'error' : ''"
+            message="App Name cannot be empty"
+          />
+        </VStack>
+        <VStack direction="column" gap="1rem" align="start">
+          <VStack gap="0.5rem">
+            <label class="app-name-label" for="app-region">Choose Region</label>
+            <VTooltip
+              title="Arcana Store uses physical storage nodes that are logically grouped by
         geography. This allows you to control the region or location where
         dApp's data assets reside, for compliance and regulatory purpose. <strong>Once a
         region has been selected it cannot be altered.</strong>"
-          >
-            <img
-              src="@/assets/iconography/info-circle-outline.svg"
-              style="cursor: pointer"
-            />
-          </VTooltip>
+            >
+              <img
+                src="@/assets/iconography/info-circle-outline.svg"
+                style="cursor: pointer"
```

Comment:
can we replace with a class.

eg: pointer {
cursor: pointer
}

Reply:
This will be fixed when we move to tailwind which is in progress